eth/logmonitor.go

```diff
@@ -0,0 +1,55 @@
+package eth
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/golang/glog"
+)
+
+type LogMonitor struct {
+	callbacks []func(j *Job)
+}
+
+func NewLogMonitor(eth LivepeerEthClient) *LogMonitor {
+	m := &LogMonitor{callbacks: make([]func(j *Job), 0)}
+
+	go func() {
+		logsCh := make(chan types.Log)
+		logsSub, err := eth.SubscribeToJobEvent(context.Background(), logsCh)
```

Comment:
Is this only useful for the `Job` event, rather than being generic to allow users to subscribe to all events?

Reply:
Hmm... I was hoping to move all log-related code in here.